src/alert/__tests__/alert.test.tsx

```diff
@@ -105,6 +105,12 @@ describe('Alert Component', () => {
       wrapper.findDismissButton()!.click();
       expect(onDismissSpy).toHaveBeenCalled();
     });
+    it('can be focused through the API', () => {
+      let ref: AlertProps.Ref | null = null;
+      render(<Alert header="Important" ref={element => (ref = element)} />);
+      ref!.focus();
```

Comment:
I think if we remove this line the test will still past, since `body` element is the `activeElement` and `Important` is in it's text content.
probably render another element, focus it and then focus the alert?